common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -155,14 +171,15 @@ div.problem {
   }
 
   .question-description {
-    @include margin(($baseline*0.75), 0);
+    font-size: 0.9em;
```

Comment:
1. The legacy way it is hard-coded all over the place.  In new CAPA, variables come from _variables.scss
2 .I realized I had a font size variable in shim file, so I am using it now.